docs/imports/rock-the-vote.md

```diff
@@ -0,0 +1,120 @@
+# Rock The Vote
+
+We import CSVs from Rock The Vote (RTV) to upsert users and their `voter-reg` posts. We previously imported this data from TurboVote (TV) in 2016, 2018.  See [VR Tech Inventory](https://docs.google.com/document/d/1xs2C3DNdD5h1j_abBrGVBNrsrxKvwn2VHDWweIEhvqc/edit?usp=sharing) for more details.
+
+## Overview
+
+The import is coded to upsert a single `voter-reg` type post for a user voter registration -- saving it to an action we set via config variable. This action is changed each election year, in order to track user registrations per election.
+
+For example, if a user registered to vote through DS in 2016, 2018, and 2020, they will have three different posts created because our import was configured for three different Action IDs in each of those election years.
+
+As another example, if a user registers to vote twice in 2020 (e.g. change of address), a single post is upserted for this year's action (two posts are not created).
+
+## Voter Registration Status
+
+Ultimately, there are 4 `post` statuses we want to capture for `voter-reg` posts for Rock the Vote (**Note** RTV doesn't have a `confirmed` status like TurboVote did):
+
+- `register-form` - User completed the registration form
+- `register-OVR` - User completed the registration form on their state's Online Voter Registration platform
+- `ineligible` - User is ineligible to register for whatever reason
+- `uncertain` - We can not be certain about this user registration status
+
+This is the import logic used to determine what the `post` status should be, looking at the following RTV columns.
+
+- If `status` is `complete` and `finish with state` is `no` --> `register-form`
+- If `status` is `complete` and `finish with state` is `yes` --> `register-OVR`
+- If `status` is any of the `step`s --> `uncertain`
+- If `status` is `rejected` --> `ineligible`
+
+Because we're pulling some of the columns into the post details, data will then be able to know if they, for example, pre-registered or why their registration was ineligible.
+
+### Status Hierarchy
+
+If RTV CSV contains multiple records _for a single user_,  we use the following hierarchy to determine which status should be reported on their Rogue post:
```

Comment:
This sort of sounds like we only use the hierarchy if the multiple rows come in on the same single CSV file - but really it's not based on if there are two rows for a user in a CSV but if a post already exists in Rogue for the `action_id` right?